data/virt_autotest/guest_params_xml_files/sles_15_sp3_64_kvm_hvm_uefi_with_power_management_x86_64.xml

```diff
@@ -24,7 +24,7 @@
     <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1</guest_installation_extra_args>
     <guest_installation_wait></guest_installation_wait>
     <guest_installation_method_others></guest_installation_method_others>
-    <guest_installation_media>http://openqa.suse.de/assets/repo/SLE-15-SP3-Full-x86_64-Build12345-Media1</guest_installation_media>
+    <guest_installation_media>http://dist.suse.de/install/SLP/SLE-15-SP3-Full-GM/x86_64/DVD1</guest_installation_media>
```

Comment:
15sp3 full media has been put into openqa.suse.de/assets/repo/fixed two days ago, do you want to switch to it?

Reply:
Done